index.html

```diff
@@ -12,8 +12,10 @@ <h1>TIC-TAC-TOE <em id="infinity">INFINITy</em></h1>
         <h3 id="board-instructions">Instructions: Select a board size by submitting a number in the field below vvvv</h3>
     </div>
     <form id='board-size-selection'>
-        <input type="number" min="3" max="25" placeholder="Choose" required>
-        <input type="submit">
+        <input type="number" min="3" max="25" name="boardSizeSelector" value=3 required>
+        <input type="text" name="otherThing" required>
```

Comment:
I put this in input (the one that says `name="otherThing"`) to show you can have multiple inputs that can all be referenced by FormData in the JS file.